core/_forms.scss

```diff
@@ -1,3 +1,6 @@
+$_form-box-shadow: inset 0 1px 3px rgba(#000, 0.06);
```

Comment:
Should we be aiming to squash all these? It's frustrating to have a bunch of Hound comments on a `bitters install`.